README.rst

```diff
@@ -32,6 +32,8 @@ Documentation for the latest **stable** version of cirq (what you get if you ``p
 - `Documentation <https://quantumai.google/cirq>`_
 - `Tutorials <https://quantumai.google/cirq/tutorials>`_
 
+Run the interactive Jupyter Notebooks in `Colab <https://colab.research.google.com/github/quantumlib/Cirq>`_.
```

Comment:
```suggestion
For a comprehensive list all of the interactive Jupyter Notebooks in our repo (including the ones not yet published to the site) open our repo in `Colab <https://colab.research.google.com/github/quantumlib/Cirq>`_.
```

Reply:
I would reword it this way, so that it's clear what users are clicking on. Can you update it please? and then I'll merge it.

Reply:
Done.